src/Microsoft.DotNet.Darc/DarcLib/LocalGitClient.cs

```diff
@@ -140,12 +138,17 @@ public async Task CommitFilesAsync(List<GitFile> filesToCommit, string repoUri,
     ///     - If no setting, or if auto, then determine whether incoming content differs in line ends vs. the
     ///       OS setting, and replace if needed.
     /// </remarks>
-    private string NormalizeLineEndings(string filePath, string content)
+    private async Task<string> NormalizeLineEndings(string repoPath, string filePath, string content)
```

Comment:
Same goes for these newly async methods. The repo isn't super consistent about FooAsync usage, but overall I've tried to go through and rename methods as I see them.